example/browser/index.js

```diff
@@ -23,64 +23,91 @@ await Exceptionless.startup(c => {
     myPassword: "123456",
     customValue: "Password",
     value: {
-      Password: "123456"
-    }
+      Password: "123456",
+    },
   };
 
   c.defaultTags.push("Example", "JavaScript");
-  c.settings["@@error:MediaError"] = "Off"
+  c.settings["@@error:MediaError"] = "Off";
 });
 
 document.addEventListener("DOMContentLoaded", () => {
   const elements = document.querySelectorAll(".submit-log");
   for (const element of elements) {
     element.addEventListener("click", (event) => {
       const level = event.target.attributes["data-level"]?.value;
-      Exceptionless.submitLog("sendEvents", `This is a log message with level: ${level || "<no log level>"}`, level);
```

Comment:
I think you have a plugin that's doing a bunch of formatting on new lines and trailing commas? Eric's vs code was doing the same thing. We need to adjust our prettify settings so this doesn't happen (I haven't been able to reproduce myself). Can we revert some of this whitespace changes. It makes it a bit harder to read.